### ISSUE TITLE
TextBox AcceptsReturn="False" is not working

### ISSUE THREAD

**Issue by deepaksm1** [wsick/Fayde#173]
TextBox AcceptsReturn="False" is not working. If we press enter key inside textbox it moves to next line.
text box should not allow new line characters.

```
  <TextBox Grid.Row="0"
                 VerticalAlignment="Center"
                 AcceptsReturn="False"                 
                 Width="400"
                 Margin="0,5"                 
                 TextWrapping="NoWrap"/>
```

**Comment by deepaksm1** [comment]
AcceptsReturn working in Firefox, not working in IE11. I am using IE 11 Version: 11.0.9600.17959

**Comment by BSick7** [comment]
Microsoft changed IE11's appName to Netscape.  This caused Fayde to use the wrong key interop.

**Comment by BSick7** [comment]
Retracting last statement, Fayde looks for IE11.  Looking deeper to see if key interactions are different in IE11.

**Comment by BSick7** [comment]
Fixed in v0.16.55.
